### DISCUSSION TITLE
Error when click on the external application button

### DISCUSSION THREAD

**Discussion by vamsi345** [Nice3point/RevitTemplates#29] [Q&A] [answered]
Hello, 
I apologize for my poor grammar. I have created the solution and project using the templates as documented in wiki. But when I'm clicking on the button in revit, I'm getting an error. I have checked all the dependencies too which are properly installed. I can't understand why I'm getting that error. I would be looking forward to feedback and get me move on with using templates to create revit addins
<img width="631" alt="Capture" src="https://user-images.githubusercontent.com/44260642/196198415-b2fd66b6-290e-466e-97bc-d7707a365621.PNG">

**Comment by Nice3point** [comment, marked as answer]
Hi, what version of templates are you using? In version 2.4.2 this problem should be fixed

**Reply by IlyaNavodkin** [reply]
use 2.4.2 , its work!

**Reply by vamsi345** [reply]
Thanks for the help! I have updated the templates, and now it is working.
